### PR TITLE
Teleoperation

### PR DESCRIPTION
**Pull request by dalinel**
Teleoperation was suppressed to be put in an other repository

* added and modified some functions to be used in ros_control
* updated yaml weights
* fixed installed when no robot_dart

### CODE REVIEW

File: CMakeCache.txt [hunk 1]
```diff
@@ -0,0 +1,38 @@
+# This is the CMakeCache file.
```
Comment (costashatz) [inline CMakeCache.txt:1]: I am sure you do not want to push this to master...

File: CMakeFiles/cmake.check_cache [hunk 2]
```diff
@@ -0,0 +1 @@
+# This file is generated by cmake for dependency checking of the CMakeCache.txt file
```
Comment (costashatz) [inline CMakeFiles/cmake.check_cache:1]: This also should not be pushed to master..

File: CMakeCache.txt (hunk 1 above)
Comment (costashatz) [reply CMakeCache.txt:1]: I mean the whole file..

**Comment by jbmouret** [comment]
OK for this PL. @dalinel should we merge?

**Comment by dalinel** [comment]
Ok perfect I will merge

File: CMakeCache.txt (hunk 1 above)
Comment (jbmouret) [reply CMakeCache.txt]: Good catch!

**Pull request merged**